cookbooks/nodejs/recipes/yarn.rb

```diff
@@ -0,0 +1,16 @@
+execute "add key from yarn repository" do
+  command "curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -"
+  not_if 'apt-key adv --list-public-key --with-fingerprint --with-colons | grep Yarn -q'
+end
+
+file "apt source yarn" do
```

Comment:
Can't we use https://docs.chef.io/resource_apt_repository.html instead of the `file` and `execute` (above) resources?

Reply:
I think we could. `apt-key adv --list-public-key --with-fingerprint --with-colons` is the same command on lib/chef/provider/apt_repository.rb.

Reply:
Updated